src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,42 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] ans = new int[matrix.length];
```

Comment:
```suggestion
        int[] diagonal = new int[matrix.length];
```
`ans` - shortened form answer, not really understandable name